client/src/components/layouts/global.css

```
@@ -656,3 +656,7 @@ pre code {
 .col-xs-4 {
   width: 33.33333333%;
 }
+
+.annual-donation-alert p {
+  margin: 0;
```

Let's be specific here. We only want to remove the bottom margin.

```suggestion
  margin-bottom: 0;
```